packages/backend-modules/documents/graphql/resolvers/Document.js

```diff
@@ -149,23 +142,23 @@ module.exports = {
       children.some((v, i) => v.data.id === startCursor && i > firstIndex)
 
     if (doc._all) {
-      // add content hash before mutating children by resolving
-      processContentHashing(doc.type, { children: nodes })
```

Comment:
we don't need this?

Reply:
No we have not figured out when this is used in the Frontend. Also you changed the function signature of `processContentHashing` a few months back from `processContentHashing(docType, content)` to `processContentHashing(content)` but did not update all the call sites so the function effectively did nothing for the last few months.